doc/getting_started/build/build_against_python_env.md

```diff
@@ -26,44 +26,28 @@ Create a new build folder and build DLite against Python 3.9
     workon dlite39
     mkdir build-dlite39
     cd build-dlite39
-    cmake -DCMAKE_INSTALL_PREFIX=$WORKON_HOME/dlite39 -DPython3_FIND_VIRTUALENV=ONLY -DPYTHON_VERSION=3.9 ..
+    cmake -DCMAKE_INSTALL_PREFIX=$VIRTUAL_ENV -DPython3_FIND_VIRTUALENV=ONLY -DPYTHON_VERSION=3.9 ..
     cmake --build .
     cmake --install .
-    ctest
 
 If the `-DPYTHON_VERSION=3.9` option to CMake is not sufficient to get CMake to compile against the correct libraries, you can make a copy of `cmake/build-python3.7.cmake` and configure CMake with
```

Comment:
```suggestion
If the `-DPYTHON_VERSION=3.9` option to CMake is not sufficient to get CMake to compile against the correct libraries, you can make a copy of `cmake/build-python3.9.cmake` and configure CMake with
```

Reply:
The cmake configuration example file we have is for Python 3.7. I agree that it might be confusing that we suddenly speak about another version...

Reply:
OK.